docs/admin/authentication.md

```diff
@@ -34,8 +34,8 @@ or be treated as an anonymous user.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
```

Comment:
Done. I hope GitHub will pick up the updated change. I also rebased on the current master.